src/locales/en/common.json

```diff
@@ -1327,8 +1327,8 @@
     "failed": "Failed",
     "fees": "Network fees",
     "noFees": "No fee",
-    "from": "From ({{count}})",
-    "to": "To ({{count}})",
```

Comment:
@juan-cortes These are still visible on non-English languages

Reply:
Indeed, we update the English one and it will be propagated through crowdin translations, we don't handle the translations manually.

Reply:
Only the en file is the source indeed. Other will be updated by crowdin.